tests/test_rados_parquet_job.py

```diff
@@ -40,7 +40,10 @@
     client = Client(cluster)
 
     # run on parquet files in cephfs (with pushdown)
-    executor_args = {"client": client, "ceph_config_path": "/etc/ceph/ceph.conf"}
+    executor_args = {
+        "client": client,
+        "ceph_config_path": "/tmp/testradosparquetjob/ceph.conf",
```

Comment:
@JayjeetAtGithub with Skyhook 0.2.0 do we still need to have explicitly passing `ceph.conf` as executor args?
At coffea-casa, we couldn't put `ceph.conf` in Docker container (security) and we simply mounted Skyhook available through CephFS using kubernetes. Using `ceph-fuse` is also not possible as well mount because we don't have root access in containers.

cc: @bbockelm @jthiltges @gattebury

Reply:
@oshadura @lgray I am done doing my changes. We can close this PR and work on the ceph.conf exposure problem separately. This PR mostly fixes a fragile CI script and updates the Skyhook docker image.